packages/block-library/src/comment-date/edit.js

```diff
@@ -60,21 +60,17 @@ export default function Edit( {
 	);
 
 	if ( ! commentId || ! date ) {
-		return (
-			<>
-				{ inspectorControls }
-				<div { ...blockProps }>
-					<time>{ _x( 'Comment Date', 'block title' ) }</time>
-				</div>
-			</>
-		);
+		date = _x( 'Comment Date', 'block title' );
 	}
 
-	let commentDate = (
-		<time dateTime={ dateI18n( 'c', date ) }>
-			{ dateI18n( format || siteFormat, date ) }
-		</time>
-	);
+	let commentDate =
+		date instanceof Date ? (
+			<time dateTime={ dateI18n( 'c', date ) }>
+				{ dateI18n( format || siteFormat, date ) }
+			</time>
+		) : (
+			date
```

Comment:
Isn't this a regression? IIUC, #41631 wrapped the "Comment date" placeholder [in a `<time />` tag](https://github.com/WordPress/gutenberg/pull/41631/files#diff-72eb391adcfe597da5b6afc8e495009af8bd546a2d2ab859b15509f83a108eb3R67) so that styling would be consistent with the frontend.

Reply:
```suggestion
			<time>{ date }</time>
```